.travis/build_deb.sh

```diff
@@ -49,16 +57,61 @@ EOF
 cat debian/changelog # debug output
 cat debian/changelog.old >> debian/changelog
 
-# build unsigned packages
-DEBUILD_OPTS+=" -eDEB_BUILD_OPTIONS=parallel=${JOBS} -us -uc -j${JOBS} -b"
+# build unsigned packages and sources on amd64
+DEBUILD_OPTS+=" -eDEB_BUILD_OPTIONS=parallel=${JOBS} -us -uc -j${JOBS}"
+# the rest will be binaries only
+if [[ ${TAG} != *"64"* ]]; then
+    DEBUILD_OPTS+=" -b"
```

Comment:
If this were `-B` instead of `-b`, [lines 112-116 below][1] might be unnecessary.

[1]: https://github.com/machinekit/machinekit/pull/817/files#diff-fe9b5219d0d614747419489fbf459234R112